src/task_types/resmoke_tasks.rs

```diff
@@ -505,18 +505,6 @@ impl GenResmokeTaskServiceImpl {
                     is_enterprise: params.is_enterprise,
                 });
             }
-            // Add a `_misc` sub-task to the list of tasks.
```

Comment:
We aren't currently running these in the build-variant level generation. And adding them is causing some problems that need to be investigated. I'm removing their inclusion for now (which is the same as our current behavior) and opened https://jira.mongodb.org/browse/DAG-1794 to investigate further.